docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+
+To learn how to expose a service, go to [this](./apix-01-expose-service-apigateway.md) tutorial.
+
+## Prerequisites
+
+If you use a cluster not managed by Gardener, install the required components manually. Follow these steps:
+
+1. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+2. Download the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) projects locally.
+
+3. Enable vertical Pod autoscaling on your cluster. For example, for Google Cloud Platform, run:
+
+   ```bash
+   gcloud beta container clusters update {PROJECT_NAME} --enable-vertical-pod-autoscaling
+   ```
+
+4. Install the `external-dns-management` component in your Namespace:
+
+   ```bash
+   helm install external-dns {PATH_TO_EXTERNAL_DNS_MANAGEMENT}/charts/external-dns-management --namespace={NAMESPACE_NAME} --set configuration.identifier=external-dns-identifier
+   ```
+
+5. Install the `cert-management` component in your Namespace:
+
+   ```bash
+   helm install cert-manager {PATH_TO_CERT_MANAGEMENT}/charts/cert-management --namespace={NAMESPACE_NAME} --set configuration.identifier=cert-manager-identifier
+   ```
+
+When you meet the prerequisites, go directly to step 2.
+
+## Steps
+
+Follow these steps to set up your custom domain and prepare a certificate required to expose a service.
+
+1. Install Kyma 2.0 or higher.
+
+2. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+3. Create a Secret containing credentails for your DNS cloud service provider account. See the [official External DNS Management docuemntation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines. Once the YAML file with the required parameters is ready, create a Secret Custom Resource (CR). Run the following command:
+
+   ```bash
+   kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
+   ```
+
+4. Set up the `external-dns-management` component.
+
+- Create a DNSProvider CR. See the following example and modify values for the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters. As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
```

Comment:
This annotation makes it to NOT work on GKE. The object is ignored. It's probably something to trigger the controller on `seed` cluster.

Reply:
In our docs I added one additional comment: `# Uncomment when using on Gardener cluster` - it is missing here. Totally agree.

Reply:
You also said that it probably doesn't to any harm to have the annotation in any case ;) I will change it to optional.